beacon-chain/chaintest/backend/setup_db.go

```diff
@@ -0,0 +1,32 @@
+package backend
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/prysmaticlabs/prysm/beacon-chain/db"
+)
+
+// setupDB instantiates and returns a simulated backend BeaconDB instance.
+func setupDB() (*db.BeaconDB, error) {
+	path := path.Join(os.TempDir(), "/simulateddb")
```

Comment:
Will this cause issues if you are running tests in parallel? 
Maybe a random named directory would be ideal.